qanta/wikipedia/wikification.py

```diff
@@ -12,6 +12,7 @@ def main():
     parser.add_argument("--output_directory", type=str,
                         default="output/wikifier/data/input/",
                         help="Where we write output file")
+    parser.add_argument("--database", type=int, default=1, help="1 to load questions, 0 for no")
```

Comment:
The relevant code is here: https://github.com/Pinafore/qb/blob/8724685c9954a4ab862f2bc7c037ad8c0a7a2d64/util/wikification.py

It was basically used to control whether to read pages from the db or from the expo output. Since 1) the expo code hasn't been added back in yet and 2) that is the wrong way to go about this, I think its better to delete the parameter altogether (which is the fix on master)